src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+        <!DOCTYPE hibernate-configuration PUBLIC "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+                "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+    <session-factory>
+        <property name ="hibernate.dialect">org.hibernate.dialect.H2Dialect</property>
+        <property name="hibernate.show_sql">false</property>
+        <property name="format_sql">false</property>
+
+        <property name="hibernate.connection.driver_class">org.h2.Driver</property>
+        <property name="hibernate.connection.url">jdbc:h2:~/todoly_db</property>
+        <property name="hibernate.connection.password">sa</property>
+        <property name="hibernate.connection.username">sa</property>
+
+        <property name="hibernate.hbm2ddl.auto">update</property>
+        <property name="current_session_context_class">thread</property>
+        <mapping class="dal.fatih.todoly.Task"/>
+        <mapping resource="named-queries.hbm.xml"/>
+    </session-factory>
+</hibernate-configuration>
```

Comment:
Delete this entire file. It is not needed since persistence.xml already has all the config defined in this file.